keyboards/adafruit/macropad/readme.md

```diff
@@ -0,0 +1,29 @@
+# Adafruit MacroPad RP2040
+
+![AdafruitMacropad](https://i.imgur.com/dSBSwcJ.jpeg)
+
+A RP2040-powered Macropad with a 3x4 layout.
+
+- Keyboard Maintainer: [Jpe230](https://github.com/jpe230/) 
+- Hardware Supported: Adafruit MacroPad RP2040
+- Hardware Availability: [Barebones kit](https://www.adafruit.com/product/5100) [Starter Kit](https://www.adafruit.com/product/5128)
+
+Make example for this board (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```